supplier_product/models.py

```diff
@@ -0,0 +1,32 @@
+from django.db import models
+from django.core.validators import MinValueValidator
+from product.models import Product
+from supplier.models import Supplier
+
+
+class SupplierProduct(models.Model):
+    supplier_product_id = models.IntegerField(primary_key=True)
```

Comment:
Why did you reput the supplier_product_id since django generate auto field that has a name id,
in this logic we need to each time to generate id for the supplier product other than django creating a one for us and this will be more complicated

Reply:
I put it since we have it in the erd .
Are you saying that i should delete this line ?

Reply:
As I mentioned [here](https://github.com/redhat-beyond/BuyTogether/pull/38#discussion_r1052195590), if you want it to be set automatically by Django, you can write:
`supplier_product_id = models.AutoField(primary_key=True)`
That way you can keep the name `supplier_product_id` and still have Django handle its values for you. You can read more about it [here](https://docs.djangoproject.com/en/4.1/ref/models/fields/#autofield).

Reply:
changed to : supplier_product_id = models.AutoField(primary_key=True)